packages/dds/tree/src/shared-tree/sharedTree.ts

```diff
@@ -275,11 +275,12 @@ class SharedTree
 
 	public fork(): ISharedTreeFork {
 		const anchors = new AnchorSet();
+		const schema = this.storedSchema.inner.clone();
```

Comment:
does editing schema on branches actually work? DO we have tests for it? Does it work with pull correctly?

Reply:
No, it's currently broken when you merge; the base view doesn't get updated by the child view. Once schema changes are part of the op/change stream and generate a delta, then it should work assuming it gets hooked up to that delta like it is the current delta. I'm not sure if that's still the approach you intend to take with schema changes, though.